entries/4/4cd.edu.json

```
@@ -0,0 +1,18 @@
+{
+    "Contra Costa Community College District": {
+        "domain": "4cd.edu",
+        "url": "https://m.4cd.edu",
+        "img": "4cd.edu.png",
+        "tfa": [
+            "sms",
+            "call",
+            "totp",
+            "u2f"
+        ],
+        "documentation": "https://4cd-doit.atlassian.net/wiki/spaces/ITPU/pages/2320465929/Setup+Multi-Factor+Authentication+MFA",
+        "keywords": [
+            "education",
+            "universities"
```

Either `education` or `universities` should be used.
```suggestion
            "universities"
```